data/abi/abi_encode.go

```diff
@@ -25,6 +25,23 @@ import (
 	"strings"
 )
 
+// bigIntToBytes casts non-negative big integer to byte slice with specific byte length
+// NOTE: THIS IS A WORKAROUND FOR `fillBytes` METHOD BEFORE GOLANG 1.15+
+//       SHOULD BE REMOVED AFTER WE MOVE TO HIGHER VERSION
```

Comment:
nit: putting deprecated in the comment here triggers some IDEs (like goland) to cross out the method.
```suggestion
// bigIntToBytes casts non-negative big integer to byte slice with specific byte length
// DEPRECATED: THIS IS A WORKAROUND FOR `fillBytes` METHOD BEFORE GOLANG 1.15+
//       SHOULD BE REMOVED AFTER WE MOVE TO HIGHER VERSION
```